contents/product-engineers/activation-metrics.mdx

```diff
@@ -0,0 +1,258 @@
+
+At PostHog, we look for a set of actions that mean someone is much more likely to be using the product 3 months later. In the case of [session replay](/session-replay), after analyzing multiple potential metrics, we found that these were watching 5 or more replays and setting a replay filter at least once.
+
+This definition solves the trickiest part of defining an activation metric: it could be _anything_. Does a pageview mean someone is activated? Does completing onboarding? What about creating their first object? Depending on your product, any of these could be it. Their impact on retention makes it clear. 
+
+## How did we find _our_ activation metric?
+
+How did we settle on `watching 5 or more replays and setting a replay filter at least once`? We tested a lot of different metrics by:
+
+1. Creating a big list of events that we thought might "hook" users into using our product.
+
+2. Creating groups of 3-5 events that you can test together. We aimed for 5-10 different groupings, including the number of times the event occurs (e.g. watched 5 replays vs. watched 1 replay). Sometimes users have to do something more than once to reach their "ah-ha" moment.
+
+3. Writing a query to test how the event groupings improve retention (see below).
+
+4. Comparing each group of events against our average retention rate for all users. We looked for a retention percentage that's nominally higher than our average retention rate.
+
+After testing multiple groupings, we found that `watching 5 or more replays and setting a replay filter at least once` was the set of events that resulted in the highest retention rate, meaning it was our activation metric.
+
+## Our query for testing potential activation metrics
+
+How can you recreate this process for yourself? Creating the grouping of potential activation events is easy, but the retention testing query is the tricky part. 
+
+To help you out, we've included the exact query we used below. If you use PostHog, you can use our same exact query to explore your data and find your activation metric. First, I'll break the query into parts to explain what's happening, then I'll provide the whole thing in one chunk at the end.
+
+> **Note:** This query gets retention at the **group** level using [group analytics](/docs/product-analytics/group-analytics). This means that anyone in the group (eg. customer, organization [what we use], work unit - whatever your groups are) can perform the event and it counts towards activation. If you want to get retention at the **person** level, you can use the same query but replace group references with person ones like changing `group_id` to `distinct_id`.
+
+### Part 1: Get the groups who started with your product, broken down by month
```

Comment:
It's probably easier to just say "company" here instead of "groups", since its more intuitive that way + most companies will be b2b sass. 
We can also mention what to do if you're b2c